src/hapi/pipelines/database/idps.py

```diff
@@ -81,9 +81,21 @@ def populate(self) -> None:
                         )
                         add_message(errors, dataset_name, text)
                         continue
+                    provider_admin1_name = ""
+                    provider_admin2_name = ""
+                    if "#adm1+name" in hxl_tags:
+                        provider_admin1_name = row[
+                            hxl_tags.index("#adm1+name")
+                        ]
+                    if "#adm2+name" in hxl_tags:
+                        provider_admin2_name = row[
+                            hxl_tags.index("#adm2+name")
+                        ]
```

Comment:
Again, suggest using if else and set provider_admin1/2_name = "" in the else part. I notice similar code in operational_presence - could this be a utility function?